packages/dappkit/README.md

```diff
@@ -66,34 +66,6 @@ Once you have the account address, you can make calls against your own smart con
   this.setState({ cUSDBalance, isLoadingBalance: false })
 ```
 
-## Get Contact List Addresses
```

Comment:
Lookups are no longer feasible in dappkit given the quota-restricted salt queries required. I'm removing this stuff until we figure out some alternative.

Reply:
Can you also remove it from our docs or at least ensure that @nityas or @critesjosh are aware to do so?